webv2/web.go

```diff
@@ -90,6 +92,27 @@ type driverConfig struct {
 	Password string `json:"Password"`
 }
 
+type sessionSummary struct {
+	DatabaseType      string
+	ConnectionDetail  string
+	SourceTableCount  int
+	SpannerTableCount int
+	SourceIndexCount  int
+	SpannerIndexCount int
+	ConnectionType    string
+}
+
+type migrationDetails struct {
+	TargetDetails targetDetails `json:"TargetDetails"`
+	MigrationMode string        `json:MigrationMode`
+	MigrationType string        `json:MigrationType`
+}
+
+type targetDetails struct {
+	TargetDB     string `json:"TargetDB"`
+	StreamingCfg string `json:"StreamingConfig"`
+}
+
 // databaseConnection creates connection with database when using
```

Comment:
Is this the correct place for these structs?
Consider if they are local to `web.go` or if they will have future usability in other locations.

Reply:
structs defined here can be used within package webv2. and they have use limited to web.go